README.md

```diff
@@ -107,7 +107,7 @@ There are multiple options for installing WebSocat. From easy to hard:
 [MacPorts](https://www.macports.org): `sudo port install websocat`
 
 ## From source
-* Install the [Rust toolchain](https://rustup.rs/) and do `cargo install --features=ssl websocat`. If something fails with a `-sys` crate, try without `--features=ssl`;
```

Comment:
Suggestion to try without `--features=ssl` is also no longer relevant and should be replaced by `--no-default-features` or something.

Reply:
Sorry, but I don't understand. By default we should want to do a default install. Which already includes `ssl` and doesn't require the complexity of a `--no-default`.

Reply:
OK, think I understand. See last commit.